test_scripts/Defects/5_0/2454_SDL_does_not_clear_app_info_after_MASTER_RESET.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2454
+--
+-- Reproduction Steps:
+-- HMI and SDL are started.
+-- Register and activate Application1.
+-- Register and activate Application2.
+-- From HMI send notification OnExitAllApplication{reason = MASTER_RESET} to SDL.
+
+-- Expected Behavior:
+-- All SDL data are cleaned and reset.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require("user_modules/script_runner")
+local common = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+local function HMISendToSDL_MASTER_RESET()
+	common.getHMIConnection():SendNotification("BasicCommunication.OnExitAllApplications",
+		  { reason = "MASTER_RESET" })
+	common.getMobileSession(1):ExpectNotification("OnAppInterfaceUnregistered",
+          { reason = "MASTER_RESET" })
+	common.getMobileSession(2):ExpectNotification("OnAppInterfaceUnregistered",
+          { reason = "MASTER_RESET" })
+		  :ValidIf(function()
+                local app_info_table = utils.jsonFileToTable(commonPreconditions:GetPathToSDL()  .. "app_info.dat")
+                local resumption_data = app_info_table.resumption.resume_app_list
+                if next(resumption_data) == nil then
+                    return true
+                else
+                    return false, "Resumption data is not cleared after MASTER_RESET"
+                end
+            end)
+    common.getHMIConnection():ExpectNotification("BasicCommunication.OnSDLClose",{})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App1", common.registerApp, { 1 })
+runner.Step("Activate App1", common.activateApp, { 1 })
+runner.Step("PTU", common.policyTableUpdate)
+runner.Step("Register App2", common.registerApp, { 2 })
+runner.Step("Activate App2", common.activateApp, { 2 })
+runner.Step("PTU", common.policyTableUpdate)
+
+runner.Title("Test")
+
```

Comment:
During script execution several times was detected that script can be sometimes passed because resumption is not saved to app_info.dat yet, so add test case 
```
local function waitUntilResumptionDataIsStored()
  utils.wait(10000)
end
```
and call it before MASTER_RESET to make sure that resumption data are saved.

Reply:
@GetmanetsIrina Added in c2b64ef205285ab4a2c50087f0c840a25f5f21c6.